usr.sbin/daemon/daemon.c

```diff
@@ -413,12 +406,23 @@ main(int argc, char *argv[])
                         }
 		}
 		execvp(argv[0], argv);
-		/*
-		 * execvp() failed -- report the error. The child is
-		 * now running, so the exit status doesn't matter.
-		 */
+		/* execvp() failed - report error and exit this process */
 		err(1, "%s", argv[0]);
 	}
+
+	/* else: pid > 0
```

Comment:
Comment starts on next line for multiline

Reply:
fixed